### ISSUE TITLE
The output dimensions of Maxpool2D do not correspond

### ISSUE THREAD

**Issue by GanchengZhu** [PINTO0309/onnx2tf#79]
### Issue Type

Others

### onnx2tf version number

1.3.6

### onnx version number

1.12.0

### tensorflow version number

2.10.0

### Download URL for ONNX

INFO: onnx_op_type: MaxPool onnx_op_name: MaxPool_346
INFO:  input_name.1: 645 shape: [1, 64, 22, 22] dtype: float32
INFO:  output_name.1: 646 shape: [1, 64, 10, 10] dtype: float32
INFO: tf_op_type: max_pool_v2
INFO:  input.1.input: name: tf.nn.relu_16/Relu:0 shape: (1, 64, 22, 22) dtype: <dtype: 'float32'> 
INFO:  input.2.filters: 
INFO:  input.3.kernel_shape: val: [3, 3] 
INFO:  input.4.strides: val: [2, 2] 
INFO:  input.5.dilations: val: [1, 1] 
INFO:  input.6.padding: val: VALID 
INFO:  input.7.ceil_mode: val: False 
INFO:  output.1.output: name: tf.nn.max_pool2d_5/MaxPool2d:0 shape: (1, 31, 10, 22) dtype: <dtype: 'float32'> 

### Parameter Replacement JSON

```json
{
  "format_version": 1,
  "operations": [
    {
      "op_name": "Reshape_54",
      "param_target": "outputs",
      "param_name": "173",
      "post_process_transpose_perm": [
        0,
        2,
        3,
        1
      ]
    },
    {
      "op_name": "Reshape_73",
      "param_target": "outputs",
      "param_name": "205",
      "post_process_transpose_perm": [
        0,
        2,
        3,
        1
      ]
    },
    {
      "op_name": "Reshape_92",
      "param_target": "outputs",
      "param_name": "237",
      "post_process_transpose_perm": [
        0,
        2,
        3,
        1
      ]
    },
    {
      "op_name": "Relu_306",
      "param_target": "outputs",
      "param_name": "576",
      "post_process_transpose_perm": [
        0,
        2,
        3,
        1
      ]
    },
    {
      "op_name": "Relu_150",
      "param_target": "outputs",
      "param_name": "320",
      "post_process_transpose_perm": [
        0,
        2,
        3,
        1
      ]
    }
  ]
}
```

### Description

In onnx model, the output of MaxPool2D is [1, 64, 10, 10], but onnx2tf output (1, 31, 10, 22). I don't know what went wrong. Maybe should I transpose the input of  MaxPool2D?

**Comment by PINTO0309** [comment]
I will close this issue in half a day if you do not share the onnx file with me.
